code/analysis/analysis_civil/figures.do

```diff
@@ -593,13 +624,22 @@ if `event' == 1 {
 		xtitle("Years relative to treatment", size(medium))
 		ytitle("Effect of MeToo on win", size(medium))
 		`xlabel'
+		text(.44 3 "ATT (Female - Male): `att_did'", size(medsmall) color(black))
+		text(.55 2.2 "ATT (Male): `att'", size(medsmall) color(black))
```

Comment:
let's change this to ATT (Women) and that's att_f and ATT (Men) and that's att

Reply:
rather than displaying the difference (att_did) i'd like to just display the two genders